packages/perseus/src/widgets/__tests__/__snapshots__/graded-group-set-jipt_test.jsx.snap

```diff
@@ -242,7 +242,10 @@ exports[`graded-group-set should render all graded groups 1`] = `
                     </div>
                   </div>
                 </span>
-                <p />
+                <p
+                  aria-live="polite"
+                  role="status"
+                />
```

Comment:
I'm not super familiar with `aria-live`. Is there a reason we'd want this on an empty `p` tag?

Reply:
Hmmm this looks like it is just the snapshot. The main change was completed in graded-group.jsx but the aria-live lets the screen reader know when that p tag is updated, as if there are any errors the tag is populated with an error message. By adding the aria-live, it means it will alert the screen reader when the content is updated (i.e. an error message is displayed). 

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

Reply:
The ticket also gives a little more background regarding what this error message is for: 

https://khanacademy.atlassian.net/browse/LP-13081

Reply:
Oh duh it's a snapshot, my bad 🤦